cli/src/commands/account.rs

```diff
@@ -36,32 +38,110 @@ pub enum Account {
         /// Seed the RNG with a numeric value
         #[clap(short = 's', long)]
         seed: Option<String>,
+        /// Try until an address with the vanity string is found
+        #[clap(short = 'v', long)]
+        vanity: Option<String>,
     },
 }
 
 impl Account {
     pub fn parse(self) -> Result<String> {
         match self {
-            Self::New { seed } => {
-                // Recover the seed.
-                let seed = match seed {
-                    // Recover the field element deterministically.
-                    Some(seed) => Field::new(
-                        <Network as Environment>::Field::from_str(&seed).map_err(|e| anyhow!("Invalid seed - {e}"))?,
-                    ),
-                    // Sample a random field element.
-                    None => Field::rand(&mut ChaChaRng::from_entropy()),
-                };
-                // Recover the private key from the seed as a field element.
-                let private_key = PrivateKey::try_from(seed)
-                    .map_err(|_| anyhow!("Failed to convert the seed into a valid private key"))?;
-                // Construct the account.
-                let account = snarkos_account::Account::<Network>::try_from(private_key)?;
-                // Print the new Aleo account.
-                Ok(account.to_string())
+            Self::New { seed, vanity } => {
```

Comment:
(this applies to the current state as well) are we expecting variants other than `New`?

Reply:
Yes, `load`